zkitter/ui
# feat: on board interep with gh

Add support for GitHub as Interep provider.
Depends on https://github.com/zkitter/zkitterd/pull/43
## Test plan
0. Ensure in your config settings (zkitterd and ui) that the both hosts are set to `http://127.0.0.1` (not `localhost`)
1. Run zkitterd on branch `r1oga/feat/gh-auth`
2. Run zkitter ui on this branch `r1oga/feat/gh-auth`
3. Add user
4. Metamask
5. Anonymous > Next > Next
6. GitHub > OAuth screen shows up. Authorize
   <img width='400' src='https://user-images.githubusercontent.com/38692952/200184653-f4feab8f-4141-4f9f-9e6a-4d2970ecdd2d.png'>
7. Join Group screen shows up > Join Group
    <img width='400' src='https://user-images.githubusercontent.com/38692952/204668334-d7493674-34c0-48a6-8ae4-219c46533438.png'>
8. Back up your identity
9. You are now logged in incognito mode and can publish tweets
